src/pages/SupplierRequisitionPage.js

```diff
@@ -163,14 +179,22 @@ export class SupplierRequisitionPage extends React.Component {
    */
   refreshData = (newSearchTerm, newSortBy, newIsAscending) => {
     const { requisition } = this.props;
+    const { useThresholdMOS, orderType } = this.state;
 
-    this.updateDataFilters(newSearchTerm, newSortBy, newIsAscending);
-    const { searchTerm, sortBy, isAscending } = this.dataFilters;
+    if (newSearchTerm && newSortBy && newIsAscending) {
+      this.updateDataFilters(newSearchTerm, newSortBy, newIsAscending);
+    }
```

Comment:
hmm not sure why this is changed, this.updateDataFilter checks `newSearchTerm && newSortBy && newIsAscending`

Reply:
done 9095693bb8516259926361a7bf6d4ca464bdad4f